### ISSUE TITLE
Consider adding layout without message bubbles

### ISSUE THREAD

**Issue by Reinachan** [SchildiChat/schildichat-android-next#5]
I'm not a big fan of message bubbles. I'd like there to be an option to make the layout similar to old Element when bubbles are disabled (modern layout as it's sometimes referred to as). I have my doubts upstream is interested in doing this, at least for the time being, so I'm suggesting it here too. My full request can be found here
https://github.com/element-hq/element-meta/discussions/2276

Is this something you'd be interested in adding support for?

**Comment by SpiritCroc** [comment]
> Is this something you'd be interested in adding support for?

No.
In fact, SchildiChat was born 50% because of (at the time being) *lack of* message bubbles in Element. So removing them is not something we're really interested in design wise.

While I'd like to say we're open to contributions for extra features that the user can configure, such features must be easily maintainable in the long run, and adding an additional layout will likely be tricky to implement in a way to satisfy this requirement. Realistically speaking, it will cause me lots of trouble for any timeline changes upstream does, so I'd say requirement would be implementing it in a way separated enough from the upstream code to allow
1. Me to disable the code on upstream merges without any hassle, without breaking compile
2. Any contributor who really wants to PR such feature to also maintain it in the long run
3. Easy dropping of the code if it's no longer sustainable to maintain.

**Comment by Reinachan** [comment]
Fair enough. I can understand your reasoning. You can close this issue unless you want to keep it up in case someone else would pick it up. I'm not proficient in app development and am highly unlikely to PR this.
